### ISSUE TITLE
[Feature request] Open sub menu in the same position as the main menu was originally opened

### ISSUE THREAD

**Issue by Sineval** [dumbeau/AutoHotPie#57]
As someone who likes to use one large, general purpose (with nested sub menus) pie menu over multiple specialized ones, I think the current way of going into nested sub menus could use some improvements.

My idea is to, basically, remember the original opening position of the main menu and use that as a starting point for all the other sub menus. This way, navigating through nested sub menus would be less "jumpy".

**Comment by dumbeau** [comment]
I agree this could use some improvements, my best answer to this at the moment is just making submenus with larger radii. 
 Also, a slice hotkey (located at the bottom of the Functions tab) can be used on a submenu slice.  Submenus open in place when activated through a slice hotkey.

This needs more context, I can make this an option but it will only apply to a few specific activation modes.  Once anything is activated 'on-hover', menus have to be added where the mouse position will be, otherwise subsequent menus will launch prematurely.

Could you screenshot your launch settings tab and post it back here for me?  It would be convenient to what activation mode you're thinking of applying this to.

**Comment by Sineval** [comment]
Right now I use the **"Hover over submenus, release on function"** with Caps-lock as an activator, but plan to move to **"Hover over all selections"** with Mouse Forward as an activator (once issue #56 is fixed), unless you can get the #58 working, cause then I will move to whatever method has that ;)

> Once anything is activated 'on-hover', menus have to be added where the mouse position will be, otherwise subsequent menus will launch prematurely.

I assume that would be caused by the mouse cursor being already atop the sub menu slice? If so, then the solution could lie, in theory, in the behavior of the option: **"Open Pie Menu in the center of screen"**, since that already moves the mouse cursor to predefined coordinates (replace the center of the screen with Pie Menu coordinates)?
